h5py/h5t.pyx

```diff
@@ -1467,7 +1501,7 @@ cdef TypeReferenceID _c_ref(object refclass):
     raise TypeError("Unrecognized reference code")
 
 
-cpdef TypeID py_create(object dtype_in, bint logical=0):
+cpdef TypeID py_create(object dtype_in, bint logical=0, bint aligned=0):
```

Comment:
stupid question, why are we taking an `object` here, rather than a `dtype` as above?